aiida/engine/processes/functions.py

```diff
@@ -51,6 +51,8 @@ def calcfunction(function):
     >>> r.get_incoming().get_node_by_label('result').get_incoming().all_nodes()
     [4, 5]
 
+    :param callable function: The function to decorate.
```

Comment:
The parameter is called `function` so remove `callable`

Reply:
I think this is actually an alternative syntax to 
```
:param function: The function to decorate.
:type function: callable
```

I'll switch to the other one now, to be consistent.